onnx/defs/tensor/defs.cc

```diff
@@ -700,7 +700,7 @@ Given `data` tensor, pads, mode, and value.
 ONNX_OPERATOR_SCHEMA(SpaceToDepth)
     .Attr(
         "blocksize",
-        "Blocks of [blocksize, blocksize] are moved.",
+        "Blocks of [blocksize, blocksize] are moved. Default blocksize is 1.",
         AttributeProto::INT,
         OPTIONAL)
```

Comment:
replace "OPTIONAL" with static_cast<int64_t>(1) then.

Reply:
In my opinion, this should be a required field

Reply:
changed required. this is resolved.